classes/Task.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private int $customer_id;
+    private ?int $executor_id;
```

Comment:
Если свойство класса не обязательно (может принимать значение `null`), то его лучше сразу проинициализировать соответствующим значением, иначе это может вызвать ошибки
```suggestion
    private ?int $executorId = null;
```